plan/constants.py

```diff
@@ -281,8 +282,9 @@ class PlanData:
         base_unit_price=PlanPrice.TEAM_MONTHLY.value,
         benefits=[
             "Up to 10 users",
-            "Unlimited public repositories",
-            "Unlimited private repositories",
+            "Unlimited repositories",
+            "2500 repositories",
+            "Patch coverage analysis",
```

Comment:
Probably a design/FE thing, but saying "Patch coverage analysis" here might give the assumption that other plans don't have patch coverage, maybe we should explicitly list all the features that each plan has. Or on the team plan say "No project coverage" or something

Reply:
Yeaah I see what you're saying, I can see the list getting exhaustive if we write all the features, but I can see the confusion of thinking this is the only plan that features patch coverage and not others, while that's not true. We can certainly followup on copy changes later if these are not final, but I'll rope @codecovdesign and @aj-codecov here for input